cmd/completion/main.go

```diff
@@ -28,25 +36,23 @@ const (
 )
 
 var (
+	cacheTag                string
 	notaryV1URL             string
 	dockerCredentials       flaghelpers.CredentialsFlags
 	dockerCfgCredentials    flaghelpers.CredentialsFlags
 	dockerConfigCredentials flaghelpers.CredentialsFlags
 	cosignAnnotations       flaghelpers.CredentialsFlags
 	cosignRepositories      flaghelpers.CredentialsFlags
 	cosignDockerMediaTypes  flaghelpers.CredentialsFlags
-	basicGitCredentials     flaghelpers.CredentialsFlags
```

Comment:
I believe the same flags from build-init are used on completion container for simplicity and these flags were added to support those flags without erroring.

Reply:
I'm a little confused by this concern, could you expand on the 
>support those flags without erroring

Reply:
All the build-init flags are passed through to the completion. So, the completion container would error on the unknown `git` flags.

Reply:
oh I see